src/test/java/com/faforever/integration/factories/SessionFactory.java

```diff
@@ -0,0 +1,85 @@
+package com.faforever.integration.factories;
+
+import com.faforever.api.client.ClientType;
+import com.faforever.api.client.OAuthClient;
+import com.faforever.api.data.domain.Player;
+import com.faforever.api.data.domain.User;
+import com.faforever.integration.TestDatabase;
+import lombok.Data;
+import lombok.SneakyThrows;
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.Assert;
+import org.springframework.security.authentication.encoding.ShaPasswordEncoder;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.util.Base64Utils;
+
+import static junitx.framework.ComparableAssert.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class SessionFactory {
+
+  private static final String OAUTH_CLIENT_ID = "1234";
+  private static final String OAUTH_SECRET = "secret";
+
+  private static ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
`No qualifying bean of type 'org.codehaus.jackson.map.ObjectMapper' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}`